x/marker/simulation/operations.go

```diff
@@ -368,6 +371,32 @@ func SimulateMsgSetAccountData(k keeper.Keeper, args *WeightedOpsArgs) simtypes.
 	}
 }
 
+// SimulateMsgUpdateSendDenyList will update random marker with denied send addresses.
+func SimulateMsgUpdateSendDenyList(k keeper.Keeper, args *WeightedOpsArgs) simtypes.Operation {
+	return func(
+		r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simtypes.Account, chainID string,
+	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
+		msg := &types.MsgUpdateSendDenyListRequest{}
+
+		marker, signer := randomMarkerWithAccessSigner(r, ctx, k, accs, types.Access_Transfer)
+		if marker == nil {
+			return simtypes.NoOpMsg(sdk.MsgTypeURL(msg), sdk.MsgTypeURL(msg), "unable to find marker with a transfer signer"), nil, nil
+		}
+
+		rDenyAccounts := simtypes.RandomAccounts(r, 10)
+		addDenyAddresses := make([]string, len(rDenyAccounts))
```

Comment:
If you are randomly adding addresses to this list (but never removing them) have you noted that all addresses for senders are eventually blocked? What kind of simulation is intended here and have you observed/proven that it properly occurs?  

How many send operations (overall) occur, how many get blocked and what percentage of the time does this occur (not how often is this method invoked which is noted as 10 in the weights above).

I am also surprised there is not a corresponding remove deny list operation.